CMakeLists.txt

```diff
@@ -93,18 +93,11 @@ endif()
 if(ANDROID)
     # Optimize for size vs. performance for Android. The has the following implications:
     # - Replace `-O2` with `-Oz`.
-    # - Clang LTO does not support `-Oz` so keep using `-O2` for that.
     # - Add `-ffunction-sections` and `-fdata-sections`. This requires that `-Wl,-gc-sections` are used when creating
     #   the final .so file.
     # - `-fstrict-aliasing` is inherited from NDK r10e.
     # - `-fomit-frame-pointer` is inherited from NDK r10e.
-    #
-    # On some architectures char is unsigned by default. Make it signed
```

Comment:
Isn't `-fsigned-char` doing what this comment is describing?

Reply:
Right, it was a mistake to remove this line. :+1: